### DISCUSSION TITLE
Closing a pinned tab should reset it to original pinned URL

### DISCUSSION THREAD

**Discussion by cizeon** [zen-browser/desktop#2632] [Ideas]
Currently, if you close a pinned tab using the keyboard shortcut, the browser simply switch to last used tab.

It'd be great to have the pinned tab go back to the original URL it was pinned to.

Current workflow:
- Youtube root is pinned.
- The user watches various videos.
- The user presses the shortcut to close the page.
- The video still plays in the backgroup and the Zen switches to last tab.

Improved workflow:
- Youtube root is pinned.
- The user watches various videos.
- The user presses the shortcut to close the page.
- The pinned tab URL is reset to Youtube root.
- Zen could also switch to last tab.

Thank you for the great work!

**Comment by rubaboo** [comment]
You don't mind having lots of shortcuts, so just fill in this one:

![image](https://github.com/user-attachments/assets/fbfd21f1-55ac-4c60-9586-63fea54266c1)

**Reply by cizeon** [reply]
Thank you for your answer.

I find having the same shortcut as closing a tab much more convenient, and you can't assign the same shortcut twice. From my experience, when you close a pinned tab, that's you want to have the URL reset and switch to last tab open.

**Comment by Redcupss** [comment]
Look for this under Tab Management in your settings.
<img width="735" alt="Screenshot 2024-11-04 at 10 24 03" src="https://github.com/user-attachments/assets/e3a2519e-1207-454b-9225-0435bfd03a3a">

**Reply by cizeon** [reply]
This is exactly what I wanted, thank you very much!

**Reply by ColinAlexndr** [reply]
This is on start-up though, is there not a way to do this immediately? Otherwise I'd have to restart Zen everytime
